ner/src/main/java/edu/illinois/cs/cogcomp/ner/LbjTagger/Parameters.java

```diff
@@ -42,7 +42,7 @@
             "Affixes", "PreviousTag1", "PreviousTag2", "PreviousTagPatternLevel1",
             "PreviousTagPatternLevel2", "AggregateContext", "AggregateGazetteerMatches",
             "PrevTagsForContext", "PredictionsLevel1", "GazetteersFeatures", "WordEmbeddings",
-            "BrownClusterPaths", "Linkability"};
```

Comment:
Regarding the size of language-specific resources: a language can have its own Brown clusters and gazetteers. Now I only have these for Chinese and Spanish. Other languages simply use the resources for English. The size of Brown clusters for one language is about 50M, and the gazetteers I added are about 100M.